src/components/OrderBuySell/index.tsx

```diff
@@ -32,7 +31,7 @@ const Wrapper = styled.div`
 
 const OrderBuySell: React.FC = () => (
   <Wrapper>
-    <Tabs tabItems={tabItems} />
+    <Tabs tabItems={tabItems} theme={tabThemeConfig} />
```

Comment:
i would consider another prop name here as it will conflict with our `ThemeProvider` props should this ever need it